.github/workflows/release.yaml

```diff
@@ -95,6 +95,6 @@ jobs:
         generate_release_notes: true
         append_body: true
         files: |
-          infrastructure-components.yaml
+          out/infrastructure-components.yaml
           metadata.yaml
           templates/cluster-template*.yaml
```

Comment:
out or templates folder should we take this file from? can we consolidate?

Reply:
templates is the versioned directory; out it a gitignored directory.